tests/browserstack/specs/ballotMainTest.js

```diff
@@ -108,85 +108,134 @@ describe('Basic cross-platform We Vote test',  () => {
-    const voteYesButtonId = 'itemActionBarYesButton-measureItem-ballotItemSupportOpposeComment-wv02meas779-desktopVersion-wv02meas779';
-    browser.execute((id) => {
-      document.getElementById(id).click();
-    }, voteYesButtonId); // Click on Voting Yes
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    // Click on Close Pop Up of Voting Yes
-    // The very first time a Yes or No option is clicked, we open a popover that explains Friends vs. Public
-    // If you try to close this popover with "profileCloseItemActionBar" later in the script, it won't be able to find this id
-    await simpleClick('profileCloseItemActionBar');
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleClick('itemPositionStatementActionBarTextArea-wv02meas779-measureItem-desktop-fromBallotItemSupportOpposeComment-wv02meas779'); // Click on TextArea
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await clearTextInputValue('itemPositionStatementActionBarTextArea-wv02meas779-measureItem-desktop-fromBallotItemSupportOpposeComment-wv02meas779',''); // Clear Text on TextArea
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleTextInput('itemPositionStatementActionBarTextArea-wv02meas779-measureItem-desktop-fromBallotItemSupportOpposeComment-wv02meas779','Commenting in measure to check'); // Write something on TextArea
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleClick('itemPositionStatementActionBarSave-wv02meas779-measureItem-desktop-fromBallotItemSupportOpposeComment-wv02meas779'); // Click on Save Button
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    const voteNoButtonId = 'itemActionBarNoButton-measureItem-ballotItemSupportOpposeComment-wv02meas779-desktopVersion-wv02meas779';
-    browser.execute((id) => {
-      document.getElementById(id).click();
-    }, voteNoButtonId); // Click on Voting No
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleClick('itemPositionStatementActionBarEdit-wv02meas779-measureItem-desktop-fromBallotItemSupportOpposeComment-wv02meas779'); // Clicks on Edit Button
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await clearTextInputValue('itemPositionStatementActionBarTextArea-wv02meas779-measureItem-desktop-fromBallotItemSupportOpposeComment-wv02meas779',' '); // Clear Text on TextArea
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    await simpleClick('itemPositionStatementActionBarSave-wv02meas779-measureItem-desktop-fromBallotItemSupportOpposeComment-wv02meas779'); // Click on Save Button
-    await browser.pause(PAUSE_DURATION_MICROSECONDS);
-    const voteNoButton = 'itemActionBarNoButton-measureItem-ballotItemSupportOpposeComment-wv02meas779-desktopVersion-wv02meas779';
-    browser.execute((id) => {
-      document.getElementById(id).click();
-    }, voteNoButton); // Click on Voting No again (to unset)
-
+    if (isDesktopScreenSize) {  // desktop version
```

Comment:
I would recommend reverting everything below line 162. There are better strategies for conditionally switching between desktop and mobile that we can discuss.